packages/amplify-graphql-api-construct-tests/src/sql-datatabase-controller.ts

```diff
@@ -0,0 +1,172 @@
+import path from 'path';
```

Comment:
This file is result of splitting the `sql-models.test.ts`.

While splitting tests, I extracted common "db setup - tear down, etc" code into this class. and changed both tests to use it.

Logic is copied pretty much verbatim from tests, but packaged in a class in order to encapsulate workflow.